code/datums/components/space_kidnap.dm

```diff
@@ -63,3 +63,11 @@
 
 		var/obj/wisp = new /obj/effect/wisp_mobile (get_turf(pick(GLOB.voidwalker_void)))
 		victim.forceMove(wisp)
+		succesfully_kidnapped()
+
+/datum/component/space_kidnap/proc/succesfully_kidnapped()
+	var/obj/item/void_eater/eater = locate(/obj/item/void_eater) in parent
```

Comment:
Maybe I am too signal pilled but could we do this by sending a `successful space kidnap` signal on the mob which is listened to by the void eater, rather than finding it directly?
Makes it easier to expand this system with other incentives later if need be

Reply:
I mean we can? I'm kinda meh on the super specific signals if it can be avoided but I'll throw it in

Reply:
"you kidnapped someone" with this component is an event and you want the weapon to respond to that event, so it makes sense to me